linkatos/activities.py

```diff
@@ -3,29 +3,107 @@
 from . import firebase as fb
 from . import reaction as react
 from . import message
+from . import cache as cch
 
 
 def is_empty(events):
+    """
+    Checks if a batch of Slack events is `None` or has zero elements.
+    """
     return ((events is None) or (len(events) == 0))
 
 
-def is_url(element):
-    return element is not None
+def is_usable(element, bot_id):
+    """
+    Checks if there is a new element(url) that is usable, by verifying that is
+    not `None` and that it is not from the bot.
```

Comment:
I think the terminology in this function can be improved by being more specific.